api/cloud-control/v1beta1/skrstatus_types.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type SkrStatusCondition struct {
+	Title           string   `json:"title"`
+	ObjKindGroup    string   `json:"objKindGroup"`
+	CrdKindGroup    string   `json:"crdKindGroup"`
+	BusolaKindGroup string   `json:"busolaKindGroup"`
+	Feature         string   `json:"feature"`
+	ObjName         string   `json:"objName"`
+	ObjNamespace    string   `json:"objNamespace"`
+	Filename        string   `json:"filename"`
+	Ok              bool     `json:"ok"`
+	Outcomes        []string `json:"outcomes"`
+}
+
+// SkrStatusSpec defines the desired state of SkrStatus.
+type SkrStatusSpec struct {
+	Kyma          string `json:"kyma"`
+	Provider      string `json:"provider"`
+	BrokerPlan    string `json:"brokerPlan"`
+	GlobalAccount string `json:"globalAccount"`
+	SubAccount    string `json:"subAccount"`
+	Region        string `json:"region"`
+	Shoot         string `json:"shoot"`
+
+	PastConnections        []metav1.Time `json:"pastConnections,omitempty"`
+	AverageIntervalSeconds int           `json:"averageIntervalSeconds,omitempty"`
+
+	Conditions []SkrStatusCondition `json:"conditions"`
+}
```

Comment:
Is Kyma same as KymaName in scope?
Same question for Shoot.

If so, perhaps I propose we stay consistent with naming, and follow the ones already defined in Scope:
https://github.com/dushanpantic/cloud-manager/blob/86db4016196cc0865dcff2ae51262723fc0bf5c7/api/cloud-control/v1beta1/scope_types.go#L41

Also, could all this be part of Scope or is new CRD really necessary?